src/components/FeaturesCard/FeaturesCard.js

```diff
@@ -18,14 +18,30 @@ const useStyles = makeStyles((theme) => ({
   title: {
     fontSize: "54px",
     fontWeight: "bolder",
+    [theme.breakpoints.down('sm')]: {
+      fontSize: '2rem'
+    }
+  },
+  image: {
+    [theme.breakpoints.down('sm')]: {
+      width: '100%',
+      height: '100%'
+    }
   },
 }));
 
 const FeaturesCard = (props) => {
   const classes = useStyles();
   return (
-    <section id={props.name}
-             style={{ marginTop: props.title == "Standardized Registration and Payments" ? "50px" : "200px" }}>
+    <section
+      id={props.name}
+      style={{
+        marginTop:
+          props.title == "Standardized Registration and Payments"
```

Comment:
This was a jank way to do it, honestly would recommend giving each section object a style attribute so we can just do something like: 
`style={ props.style }`

Reply:
*I originally implemented this fix lol. don't think i'm getting mad at your code or anything